lib/charms/opensearch/v0/helper_charm.py

```diff
@@ -136,6 +136,31 @@ def format_unit_name(unit: Union[Unit, str], app: App) -> str:
     return f"{unit.replace('/', '-')}.{app.short_id}"
 
 
+def all_units_names(charm: "OpenSearchBaseCharm") -> List[Unit]:
```

Comment:
Keeping this logic, although not used anymore in this code.

We need a way to recover all units' names, disregard if this is a small, large, etc deployment.